CONTRIBUTING.md

```diff
@@ -45,6 +45,19 @@ After a successful build the RAPIDS Accelerator jar will be in the `dist/target/
 You can build against different versions of the CUDA Toolkit by using one of the following profiles:
 * `-Pcuda11` (CUDA 11.0/11.1/11.2, default)
 
+### Overriding memory settings
+
+It is sometimes necessary to override default memory settings when running the test suite. This can be achieved 
+by using Java system properties to override any configuration specific that is specific to the RAPIDS Accelerator. 
+
+For example, if the GPU is being used to drive a display then there is less memory available, and it may be 
+necessary to reduce the `spark.rapids.memory.gpu.allocFraction` setting from the default value. The following 
+syntax can be used to pass Java system properties to the Maven Surefire plugin. 
+
+```shell script
+mvn verify -DargLine="-Dspark.rapids.memory.gpu.allocFraction=0.7"
```

Comment:
I understand why you want this, but it scares me.  Spark does not support this and if someone sets a config through a system property in production I'm not sure how we can debug it. Spark will not see it so it will not be logged/displayed in the typical places. We already have something that does this just for the tests but it is not well documented.

https://github.com/NVIDIA/spark-rapids/blob/247b75847bc455aad799c30bdb792a4acea7c255/tests/src/test/scala/com/nvidia/spark/rapids/SparkQueryCompareTestSuite.scala#L88-L94

Could you use that instead? and document it a little better.